src/feature.js

```diff
@@ -413,24 +413,37 @@ var feature = function (arg) {
   ////////////////////////////////////////////////////////////////////////////
   /**
    * Get/Set visibility of the feature
+   *
+   * @param {boolean|undefined} val: undefined to return the visibility, a
+   *    boolean to change the visibility.
+   * @param {boolean} direct: if true, when getting the visibility, disregard
```

Comment:
right but I could see a use-case (perhaps more complicated) where a user would like to know the visible state of a feature not w.r.t to its parent. But I agree that most likely they would just query the state w.r.t parent.